mount_linux.go

```diff
@@ -123,3 +133,19 @@ func mount(dir string, cfg *MountConfig, ready chan<- error) (*os.File, error) {
 	}
 	return dev, err
 }
+
+var DEV_FD = regexp.MustCompile("^/dev/fd/([0-9]+)$")
+
+func parseFuseFd(dir string) (int, error) {
+	matches := DEV_FD.FindAllStringSubmatch(dir, 1)
```

Comment:
Can you use `strings.HasPrefix` and `strings.TrimPrefix` instead? All the regexp does is ensure the prefix matches — the `[0-9]+` part of it is covered by `strconv.Atoi` already.

Reply:
The regexp didn't allow negative fds, so I ended up using `strconv.ParseUint` instead. Let me know how it looks.

Reply:
And thanks for the quick reply!